lib/pages/chat/events/sending_image_info_widget.dart

```diff
@@ -48,55 +52,62 @@ class SendingImageInfoWidget extends StatelessWidget {
-                strokeWidth: 2,
-                color: LinagoraRefColors.material().primary[100],
-              ),
-              Icon(
-                Icons.close,
-                color: LinagoraRefColors.material().primary[100],
-              ),
-            ]
-          ],
-        );
-      },
-      child: InkWell(
-        onTap: () => _onTap(context),
-        child: ClipRRect(
-          borderRadius: BorderRadius.circular(12.0),
-          child: Stack(
+    return Hero(
+      tag: event.eventId,
+      child: ValueListenableBuilder<double>(
+        key: ValueKey(event.eventId),
+        valueListenable: sendingFileProgressNotifier,
+        builder: (context, value, child) {
+          return Stack(
             alignment: Alignment.center,
             children: [
-              if (displayImageInfo.hasBlur)
-                SizedBox(
-                  width: MessageContentStyle.imageBubbleWidth(
-                    displayImageInfo.size.width,
-                  ),
-                  height: MessageContentStyle.imageBubbleHeight(
-                    displayImageInfo.size.height,
-                  ),
-                  child:
-                      const BlurHash(hash: MessageContentStyle.defaultBlurHash),
+              child!,
+              if (sendingFileProgressNotifier.value != 1) ...[
+                CircularProgressIndicator(
+                  strokeWidth: 2,
+                  color: LinagoraRefColors.material().primary[100],
+                ),
+                Icon(
+                  Icons.close,
+                  color: LinagoraRefColors.material().primary[100],
                 ),
-              Image.file(
-                File(matrixFile.filePath!),
-                width: displayImageInfo.size.width,
-                height: displayImageInfo.size.height,
-                cacheHeight: displayImageInfo.size.height.toInt(),
-                cacheWidth: displayImageInfo.size.width.toInt(),
-                fit: BoxFit.cover,
-                filterQuality: FilterQuality.medium,
-              )
+              ]
             ],
+          );
+        },
+        child: Material(
+          borderRadius: BorderRadius.circular(12.0),
+          child: InkWell(
+            onTap: () => _onTap(context),
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(12.0),
+              child: Stack(
+                alignment: Alignment.center,
+                children: [
+                  if (displayImageInfo.hasBlur)
+                    SizedBox(
+                      width: MessageContentStyle.imageBubbleWidth(
+                        displayImageInfo.size.width,
+                      ),
+                      height: MessageContentStyle.imageBubbleHeight(
+                        displayImageInfo.size.height,
+                      ),
+                      child: const BlurHash(
+                        hash: MessageContentStyle.defaultBlurHash,
```

Comment:
why default blur hash ?
```suggestion
                        hash:  displayImageInfo.blurhash ?? MessageContentStyle.defaultBlurHash,
```

Reply:
okay old code